docs/source/docker/integration/docker_with_datasource.rst

```diff
@@ -0,0 +1,101 @@
+
+.. _ref-docker-with-datasource:
+
+Using the docker with an S3 bucket as remote datasource.
+========================================================
+
+Introduction
+------------
+The Lightly Docker can be used with the Lightly Platform to do
+the following workloads in one single run:
+
+- stream your files directly from your AWS S3 bucket to your local machine without
+  needing to sync or download them
+- embed all images or video frames
+- sample a subset, e.g. using CORESET
+- compute the metadata of the images
+- create a dataset in the Lightly Platform from the sampled subset
+
+Advantages
+----------
+
+- You can run the whole Lightly workflow with one single command.
+- You can process videos directly without needing to extract frames.
+- Your data is streamed from your S3 bucket on the go instead of being first downloaded
+  to you local disk and then read from there. Thus you save both a lot of time and
+  disk space and can process much larger datasets.
+- You can :ref:`ref-docker-with-datasource-datapool`
+  to continuously update and improve your subsampled dataset
+  everytime new data comes into your S3 bucket.
+
+
+Requirements
+------------
+
+This recipe requires that you already have a dataset in the Lightly Platform
+configured to use the data in your AWS S3 bucket.
+
+Follow the steps on how to `create a Lightly dataset connected to your S3 bucket <https://docs.lightly.ai/getting_started/dataset_creation/dataset_creation_aws_bucket.html>`_.
```

Comment:
I would not add all the dataset creation elements here for two reasons:
- It would mean duplicating lots of stuff.
- It does not work well when we also have support for GC storage and azure storage.